source/features/more-conversation-filters.tsx

```diff
@@ -17,12 +17,11 @@ function init(): void {
 	sourceItem.after(commentsLink);
 
 	// "Subscribed" link
-	const subscriptionsLink = commentsLink.cloneNode(true);
-	subscriptionsLink.lastChild!.textContent = 'Everything you subscribed to';
-	subscriptionsLink.setAttribute('aria-checked', 'false'); // #4589
+	const subscriptionsLink = select('#filters-select-menu a:last-child')!.cloneNode(true);
+	subscriptionsLink.lastElementChild!.textContent = 'Everything you subscribed to';
 
 	const subscriptionsUrl = new URL('https://github.com/notifications/subscriptions');
-	const repositoryId = select('input[name="repository_id"]')!.value;
+	const repositoryId = select('meta[name="octolytics-dimension-repository_id"]')!.content;
```

Comment:
Worth preserving previous logic for GHE?

Reply:
Sure if you feel like it. I've been keeping both selectors wherever simple, but if the logic is too different either we keep two versions or we drop GHE.

In this case I think you would need to write:

``` 
select(meta)?.content ?? select(input)!.value
```
This is probably the most straightforward way and the easiest one to drop in the future (which should always be our priority when doing this kind of code forks)